lib/src/animation_controller.dart

```diff
@@ -39,9 +39,50 @@ class AnimationControllerValue {
   }
 }
 
-class RubberAnimationController extends Animation<double>
-  with AnimationEagerListenerMixin, AnimationLocalListenersMixin, AnimationLocalStatusListenersMixin {
+/// Pads the animation using value.clamp
+///
+/// Useful for preventing the sheet from speeding off screen
+/// without compromising too much on bounciness
+///
+/// NOTE: the [top] padding value is interpeted as a distance from the top.
+/// If you want to specify a range, use the `.fromPercentageRange` constructor
+class AnimationPadding {
```

Comment:
I like the AnimationPadding solution, but it would be great if there was  a _null_ constructor that doesn't clamp the animation at all to preserve the original behavior

Reply:
The default right now is to `clamp(0, 1)`, which is the screen height. I feel like clamping to the screen height is a better default than allowing the sheet to be flung off screen (was my original issue, if I remember correctly)

Reply:
You're right but right now there is no way to avoid the clamping... I prefer the animation that exceeds the limits but I understand that in many cases it is better to avoid it, that's why it would be great to have a "null" constructor for the padding